source/VexiiRiscv/Introduction/index.rst

```diff
@@ -52,17 +52,31 @@ Here is a list of important assumptions and things to know about :
 - In the execute pipeline, stage.up(RS1/RS2) is the value to be used, while stage.down(RS1/RS2) should not be used, as it implement the bypassing for the next stage
 - Fetch.ctrl(0) isn't persistant.
 
-About VexRiscv (not VexiiRiscv)
+Why a New Design? 
```

Comment:
> Why a New Design

Hmm maybe "new design" is too much implicit (maybe it should be more explicit)
"Why VexiiRiscv" ?